Users/coafile.rst

```diff
@@ -30,7 +30,7 @@ User-Wide and System-Wide coafile
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
```

Comment:
so firstly, in shortlog you dont need to mention where the sentence is changed, as "coafile.rst" already specifies the file that is being worked on.'

Secondly, we dont require such an explicit body, i feel like its redundant. something shorter like this is better: ``The sentence was wrong, as the settings are user wide, and is now correct.``